redisinsight/ui/src/pages/browser/components/actions/Actions.tsx

```diff
@@ -6,7 +6,7 @@ import {
   TelemetryEvent,
 } from 'uiSrc/telemetry'
 import {
-  PrimaryButton,
+  Button,
```

Comment:
nit: We also have an alias for a `SecondaryButton` that you can use :)

Reply:
Changed but I didn't like it. In general we added custom props to change the behavior and it became not clear which exactly variant we use.

Reply:
Usually, the goal of having semantics in the way we build up our design systems is to hide the "implementation details" and make the end result look consistent everywhere. 

Yes, we can always use the base components and customize them however we want, but I believe this is what we wanted to avoid when @pd-redis's introduced these semantic aliases. Anyway, we can talk about this in the future, it's not something critical/important right now :)